melody/converter.js

```diff
@@ -0,0 +1,9 @@
+const exec = require("child_process").exec;
+
+module.exports = async function(filepath) {
+  return new Promise((resolve, reject) => {
+    exec(`midi2abc ${filepath}`, function(error, stdout, stderr) {
```

Comment:
By @akellbl4:

I prefer to use arrow function when it passing as a callback, but it is always up to you.
Also, it not necessary to define an argument if you won't use it.

```
  exec(`midi2abc ${filepath}`, (_, stdout) => resolve(stdout));
```